File: c++/S01-elemental-actions/E07-multiplication-table.cpp
#include <iostream>

int main() {
	int value;
	std::cout << "\n\e[0;35m[========= MULTIPLICATION TABLE =========]\e[0m\n" << '\n';

	std::cout << "Enter a number: ";
	std::cin >> value;

	std::cout << "Multiplication table of " << value << '\n';
	std::cout << "==========================" << '\n';
	std::cout << "1 x " << value << " = " << 1 * value << '\n';
	std::cout << "2 x " << value << " = " << 2 * value << '\n';
	std::cout << "3 x " << value << " = " << 3 * value << '\n';
	std::cout << "4 x " << value << " = " << 4 * value << '\n';
	std::cout << "5 x " << value << " = " << 5 * value << '\n';
	std::cout << "6 x " << value << " = " << 6 * value << '\n';
	std::cout << "7 x " << value << " = " << 7 * value << '\n';
	std::cout << "8 x " << value << " = " << 8 * value << '\n';
	std::cout << "9 x " << value << " = " << 9 * value << '\n';

	return 0;
}

File: python/S01-elemental-actions/E07-multiplication-table.py
def main():
    print("\n\033[35m[========= MULTIPLICATION TABLE =========]\033[0m\n")

    number = int(input("Enter a number: "))

    print("\nMultiplication Table of", number)
    print("===========================")
    print(f"1  * {number} = {number * 1}")
    print(f"2  * {number} = {number * 2}")
    print(f"3  * {number} = {number * 3}")
    print(f"4  * {number} = {number * 4}")
    print(f"5  * {number} = {number * 5}")
    print(f"6  * {number} = {number * 6}")
    print(f"7  * {number} = {number * 7}")
    print(f"8  * {number} = {number * 8}")
    print(f"9  * {number} = {number * 9}")
    print(f"10 * {number} = {number * 10}")


if __name__ == "__main__":
    main()
